web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-studio.adoc

```diff
@@ -0,0 +1,282 @@
+
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met.
+
+For example, to configure `HTTP Listener`, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add Consume Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add the Consume operation for Web Service Consumer Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Web Service Consumer* and then select the *Consume* operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+In Studio, the Consume operation looks like this:
+
+.Web Service Consume Operation in Studio
+image::web-service-consumer-consume-operation.png[Web Service Consume Operation in Studio]
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. To consume a SOAP Web service operation, you need to create a configuration that points to the service that you want to consume.
+
+To configure the global element for Web Service Consumer Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the *Basic Settings* section of the *General* tab, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *Connection* section of the *General* tab, configure the following parameters:
+* *WSDL location* +
+The WSDL file URL remote or local.
+* *Service* +
+The service name.
+* *Port* +
+The port name.
+* *Address* +
+The address of the web server to dispatch requests but if the *Service* and *Port* parameters don't auto-populate the web server address, you must manually provide the correct *Address*.
+
+Note that *WSDL location*, *Service*, and *Port* parameters are mandatory for the configuration to be valid.
+
+The following example shows how to configure the Global Element to get a list of flights by airline:
```

Comment:
```suggestion
The following example shows how to configure the global element to get a list of flights by airline:
```

Reply:
This is lowercased above, but I see that the term presentation list makes a distinction between **a** global element and **the** Mule Global Element...and I wonder if it should? Maybe I need to correct the term list? What do you think?